### PR TITLE
Issue 372:  Update image paths

### LINKED ISSUE #372: Update links from the README file
There are some images paths on the README.md file that don't work properly in the Docusaurus website, so we need to fix them.

### PR DESCRIPTION
**Pull request by jorgeMarin1**
## Description
Fixes #372 

Some gifs and images were not showing up in the Github Pages website, so I had to change a few filepaths in the README.md file.

For example, I had to change from ```Qbeast spark project``` to pure markdown```Qbeast-spark```

Please review whenever you can @cdelfosse @Jiaweihu08 , thanks!

## Type of change

Documentation update. As I said, I changed a few filepaths in the main README file of this repository.

## Checklist:

- [X] Change the documentation.
- [X] Your branch is updated to the main branch (dependent changes have been merged).

### CODE REVIEW

**Review by cdelfosse — APPROVED** [review]

**Comment by jorgeMarin1** [comment]
Please don't merge this PR, I still need to make some fixes

**Comment by jorgeMarin1** [comment]
Ok I don't need to make more changes. This PR is ready to me squeashed and merged.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,5 @@
-<p align="center">
-	<img src="./docs/images/Qbeast-spark.png" alt="Qbeast spark project"/>
-</p>
+![Qbeast-spark](./docs/images/Qbeast-spark.png)
```
Comment (Jiaweihu08) [inline README.md:1]: Is the image still centered?

Comment (jorgeMarin1) [reply README.md:2]: Ok the images should be centered again

Comment (jorgeMarin1) [reply README.md:2]: When I use  this code to center an image:
```
<p align="center">
![Qbeast-spark](./docs/images/Qbeast-spark.png)
</p>
```

The image looks like this in the website: 
Screenshot 2024-07-31 at 16 39 41

But looks like this in the qbeast-spark repository:
Screenshot 2024-07-31 at 16 40 21

Comment (jorgeMarin1) [reply README.md:2]: If I use the original code from the qbeast-spark repository:

```
<p align="center">
	<img src="./docs/images/Qbeast-spark.png" alt="Qbeast spark project"/>
</p>
```

The spark logo will show up in the github repository, but will look like this in the website:

Screenshot 2024-07-31 at 16 56 41

**Comment by osopardo1** [comment]
I've checked the PR and looks good! 

Just one reminder: if you want all the paths to be absolute, there are a couple of links in the README that are missing to be changed. 

image

The quickstart and notebook.

**Comment by jorgeMarin1** [comment]
@osopardo1 I've checked what you said about the links of the quickstart and the notebook and I don't think we need to change that because it works.

The thing is that for some images when we use html to center them, depending on if we use html or markdown to show the images they either don't appear in the Docusaurus website or in our github repository.

But, for some reason I dont't understand, if we use pure html with the absoule paths to the images they seem to show in both places...

`<p align="center">
	Qbeast spark project
</p>`

**Pull request merged**